plugin-dev/Source/Sentry/Sentry_Android_UPL.xml

```diff
@@ -137,7 +141,7 @@
     <buildscriptGradleAdditions>
         <insert>
             dependencies {
-                classpath 'com.android.tools.build:gradle:3.5.4'
+                classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
That's the gradle tool itself, right? And that's a requirement for the sentry-android-gradle-plugin `4.11.0`?
Are we going to mess with user's build by messing with the gradle version that comes with the UE installation?

Reply:
Yes, the `io.sentry.android.gradle >= 3.0.0` requires `Android Gradle Plugin >= 7.0.0` according to docs.
I've tested this with a few different engine versions (including pretty old UE 4.27) and the build was successful however there's always a chance that some compatibility issues with 3rd-party plugins can arise.